spec/models/registration_spec.rb

```diff
@@ -22,4 +22,27 @@
       end
     end
   end
+  describe '#destroy_purchased_tickets' do
+    it 'destroys purchased tickets if tickets are purchased' do
+      user = build(:user)
+      conference = build(:conference)
+      registration1 = create(:registration, conference: conference, user: user)
+      create(:ticket_purchase, conference: conference, user: user)
+      expect(user.registrations.size).to eq(1)
+      expect(user.ticket_purchases.size).to eq(1)
+      registration1.destroy
+      expect(user.registrations.size).to eq(0)
+      expect(user.ticket_purchases.size).to eq(0)
+    end
+    it 'destroys no tickets if no tickets are purchased' do
```

Comment:
Please add an empty line between the test cases (usually rubocop reports about this ... grml)